### PR TITLE
`<MessageBoxFunctionalLayout/>`- using the new <Button/> instead of the deprecated one

### LINKED ISSUE #4644: `<MessageBoxFunctionalLayout />` - ButtonsHeight 'tiny' sets sizes of 'small'
# 🐛 Bug Report

### 🏗 Relevant Components
`<MessageBoxFunctionalLayout />`

...

### 😯 Current Behavior

When setting the `buttonsHeight` prop to tiny, the buttons get bigger than when setting the prop to small

### 👣 Steps to Reproduce

  - **WSR version**: `7.15`
  - **Yoshi version**: `3`

### 👀 Severity

- Critical

### PR DESCRIPTION
**Pull request by marik22312**
### 🔦 Summary

Updated Buttons class names to fix issue #4644m,
Hope it's ok

Resolves #4644

### ✅ Checklist
- 📚 Change is documented
  - [X] Story
- 🔬 Change is tested
  - [X] Component
  - [X] Visual test

### CODE REVIEW

File: src/MessageBox/FunctionalLayout/MessageBoxFunctionalLayout.driver.js [hunk 1]
```diff
@@ -38,9 +38,9 @@ const messageBoxFunctionalLayoutDriverFactory = ({ element }) => {
       element.querySelector('[data-hook="header-layout-title"]').textContent,
     getChildBySelector: selector => element.querySelector(selector),
     isCancelEnable: () =>
-      cancellationButton().className.indexOf('disabled') === -1,
+      cancellationButton().hasAttribute('disabled') !== true,
     isConfirmationEnable: () =>
-      confirmationButton().className.indexOf('disabled') === -1,
+      confirmationButton().hasAttribute('disabled') !== true,
```
Comment (EgoziE) [inline src/MessageBox/FunctionalLayout/MessageBoxFunctionalLayout.driver.js]:
```suggestion
      !confirmationButton().hasAttribute('disabled'),
```

File: src/MessageBox/FunctionalLayout/MessageBoxFunctionalLayout.driver.js [hunk 2]
```diff
@@ -38,9 +38,9 @@ const messageBoxFunctionalLayoutDriverFactory = ({ element }) => {
       element.querySelector('[data-hook="header-layout-title"]').textContent,
     getChildBySelector: selector => element.querySelector(selector),
     isCancelEnable: () =>
-      cancellationButton().className.indexOf('disabled') === -1,
+      cancellationButton().hasAttribute('disabled') !== true,
```
Comment (EgoziE) [inline src/MessageBox/FunctionalLayout/MessageBoxFunctionalLayout.driver.js]:
```suggestion
      !cancellationButton().hasAttribute('disabled'),
```

**Comment by TalZa1810** [comment]
Hi @marik22312 ,
Thanks for your contribution :)  
We've fixed the driver functions. 
Visual tests are still broken. The `skin` of the new button is missing (replaces the prop `theme`).
Further work needs to be done before merging.

**Comment by marik22312** [comment]
Hey @TalZa1810 
Thank you! hope ill have time to finish the work next week

**Comment by TalZa1810** [comment]
Hi @marik22312 :) 
Any estimated time to finish with this PR?

**Comment by marik22312** [comment]
Hi @TalZa1810 !
Unfortunately no, I'm busy with my teams tasks...

**Comment by shlomitc** [comment]
I'm closing this PR, please revive it when working on it again.
Thanks

**Pull request closed without merging**